src/main/java/cn/nukkit/block/BlockGrass.java

```diff
@@ -61,10 +62,16 @@ public boolean onActivate(Item item, Player player) {
             ObjectTallGrass.growGrass(this.getLevel(), this, new NukkitRandom());
             return true;
         } else if (item.isHoe()) {
+            if(player != null){
+                player.getLevel().addSound(player, Sound.USE_GRASS);
+            }
             item.useOn(this);
             this.getLevel().setBlock(this, Block.get(BlockID.FARMLAND));
             return true;
         } else if (item.isShovel()) {
+            if(player != null){
+                player.getLevel().addSound(player, Sound.USE_GRASS);
```

Comment:
Probably better to play the sound AFTER the action is executed for consistent responses